apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -0,0 +1,374 @@
+import React from 'react';
+
+/**
+ * Entry point to build a bundle containing a set of globals used when displaying
+ * tutorialExplorer.
+ */
+require('babel-polyfill');
+window.React = require('react');
+window.ReactDOM = require('react-dom');
+window.Radium = require('radium');
+var update = require('react-addons-update');
+
+window.TutorialExplorerManager = function (options) {
+  this.options = options;
+  var self = this;
+
+  const FilterChoice = React.createClass({
+    propTypes: {
+      onUserInput: React.PropTypes.func,
+      groupName: React.PropTypes.string,
+      name: React.PropTypes.string,
+      isCheckedInput: React.PropTypes.bool,
+      value: React.PropTypes.string,
+      selected: React.PropTypes.bool,
+      text: React.PropTypes.string
+    },
+
+    handleChange: function () {
+      this.props.onUserInput(
+        this.props.groupName,
+        this.props.name,
+        this.refs.isCheckedInput.checked
+      );
+    },
+
+    render() {
+      return (
+        <div  style={{userSelect: 'none', WebkitUserSelect: 'none', MozUserSelect: 'none', MsUserSelect: 'none'}}>
+          <label style={{fontFamily: "\"Gotham 4r\", sans-serif", fontSize: 13, paddingBottom: 0, marginBottom: 0, cursor: 'pointer'}}>
```

Comment:
Style nit:
Our approach for semi-complicated style objects has been to do something like this: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/components/progress/TeacherStageInfo.jsx#L20